src/gsi/unfill_mass_grid2.f90

```diff
@@ -381,3 +381,107 @@ subroutine unfill_mass_grid2t_drycheck(gout,nx,ny,gin,qs)
   end do
   
 end subroutine unfill_mass_grid2t_drycheck
+
+subroutine unfill_mass_grid2t_ldlkmask(gout,nx,ny,gin,landmask, &
+                            lakemask,i_howv_mask)
+!$$$  subprogram documentation block
+!                .      .    .                                       .
+! subprogram:    unfill_mass_grid2t        opposite of fill_mass_grid2
+!   prgmmr: parrish          org: np22                date: 2004-06-22
+!
+! abstract: This is the same as unfill_mass_grid2t and unfill_mass_grid2t_ldmk
+!           but screen off analysis increment over land and/or lake . 
+!
+!
+! program history log:
```

Comment:
If this is a new subroutine added to GSI, why does it have a program history going back to 2004?

Reply:
@RussTreadon-NOAA,
I updated this information. Thank you!